carbon_tracker_workflow.ipynb

```diff
@@ -0,0 +1,2510 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    carbon_tracker.validate_model(ds_na, 5, x_keys, y_key, output_path)
</pre><p>It's a shame those pycaret plots don't use scientific notation. Can you set round=7 or something to still show RMSE etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EXCITED-CO2/excited-workflow/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
round=7 works but now I wonder if there is a better way to do it, is ok for now though.